gc/base/GCExtensionsBase.cpp

```diff
@@ -182,10 +182,12 @@ MM_GCExtensionsBase::initialize(MM_EnvironmentBase* env)
 	if (J9HookInitializeInterface(getPrivateHookInterface(), OMRPORTLIB, sizeof(privateHookInterface))) {
 		goto failed;
 	}
+	INITIALIZE_HOOKDUMP(privateHookInterface);
```

Comment:
array of OMREventInfo4Dump is added in individual hookInterface by Hook generation code to avoid rumtime native memory allocation(malloc), but hookable code is not aware of specific Hook interface except J9CommonHookInterface, the macro OMRHOOK_INITIALIZE_HOOKDUMP try to pass array pointer and size from the  individual hookInterface to J9CommonHookInterface, then hookable code and javadump code can easily access OMREventInfo4Dump.